tasks/28_vowel-count/vowel-count.js

```diff
@@ -0,0 +1,6 @@
+export const getVowelCount = (str) =>{
+  let pattern = /[aeiou]/gi;
+  let vowelAmount = str.split('').filter((cur) => cur.match(pattern)).length;
```

Comment:
you can do the same thing just with match method as it will return the array of all vowels and you would need to return length of this array